recipe_scrapers/tofoo.py

```diff
@@ -14,49 +14,56 @@ def author(self):
         return "The Tofoo co."
```

Comment:
Not directly related, but I think we should be using a `StaticValueException` here (seems I missed this method when introducing those exceptions).

Reply:
Just added it here!